codex-skill/SKILL.md

```
@@ -0,0 +1,364 @@
+```
+
+**Important:** Manual JSON editing is fragile. Validate syntax after editing:
+```bash
+jq empty .ba/issues.jsonl  # Exit 0 = valid, non-zero = syntax error
+```
+
+**Future consideration:** Add `ba reclaim --force <id> --session $SESSION_ID` command to handle this safely.
+
+## Quick Reference Card
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)